### PR TITLE
feat: check output.path and output.publicPath if humanErrors are enabled

### PR DESCRIPTION
**Pull request by satazor**

### CODE REVIEW

File: lib/checkHumanErrors.js [hunk 1]
```diff
@@ -2,30 +2,52 @@
...
+    str && options.write(str);
+}
+
+function checkOutputPaths(compiler, options) {
+    const clientOutput = compiler.client.webpackConfig.output;
+    const serverOutput = compiler.server.webpackConfig.output;
+    let str = '';
+
+    if (clientOutput.path !== serverOutput.path) {
+        str += `${chalk.yellow('WARN')}: The \`output.path\` should the same in the webpack client & server configs\n`;
```
Comment (andrefgneves) [inline lib/checkHumanErrors.js]: "should _be_ the same"

File: lib/checkHumanErrors.js [hunk 2]
```diff
@@ -2,30 +2,52 @@
...
+        str += `Client output.path: ${clientOutput.path}\n`;
+        str += `Server output.path: ${serverOutput.path}\n\n`;
+    }
+    if (clientOutput.publicPath !== serverOutput.publicPath) {
+        str += `${chalk.yellow('WARN')}: The \`output.publicPath\` should the same in the webpack client & server configs\n`;
```
Comment (andrefgneves) [inline lib/checkHumanErrors.js]: "should _be_ the same"

**Review by andrefgneves — CHANGES_REQUESTED** [review]

File: lib/checkHumanErrors.js (hunk 1 above)
Comment (satazor) [reply lib/checkHumanErrors.js]: fixed

File: lib/checkHumanErrors.js (hunk 2 above)
Comment (satazor) [reply lib/checkHumanErrors.js]: fixed

**Review by andrefgneves — APPROVED** [review]

**Pull request merged**